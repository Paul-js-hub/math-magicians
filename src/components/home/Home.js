import React from 'react';
import home from './home.module.css';

const Home = () => (
  <div className="home">
    <h3 className={home.header3}>Welcome to Math Magicians</h3>
    <p className={home.paragraph}>
      Math Magicians is a webpage that is designed to provide users with a fun
      way to interact with online calculator. It also generates random quotes
      about Mathematics
    </p>
  </div>
);

export default Home;
