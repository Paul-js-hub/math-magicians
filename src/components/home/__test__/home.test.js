import Home from '../Home.js';
import renderer from 'react-test-renderer';

test('it returns a calculator component', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});