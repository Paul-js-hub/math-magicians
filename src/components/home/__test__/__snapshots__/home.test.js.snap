// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it returns a calculator component 1`] = `
<div
  className="home"
>
  <h3
    className="header3"
  >
    Welcome to Math Magicians
  </h3>
  <p
    className="paragraph"
  >
    Math Magicians is a webpage that is designed to provide users with a fun way to interact with online calculator. It also generates random quotes about Mathematics
  </p>
</div>
`;
